Testing.sln

```diff
@@ -31,6 +31,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Framework.TestHos
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.TestHost.Client", "src\Microsoft.Framework.TestHost.Client\Microsoft.Framework.TestHost.Client.xproj", "{90C1A9CE-E402-4206-8194-03A844435FC3}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.WebEncoders.Testing", "src\Microsoft.Framework.WebEncoders.Testing\Microsoft.Framework.WebEncoders.Testing.xproj", "{12732002-3304-4741-9914-8FCCB2C28193}"
```

Comment:
why a new project instead of 2 new classes in src\Microsoft.AspNet.Testing?  the src\Microsoft.Framework.Logging.Testing project involves a few more classes but I could go either way here.  would like to be sure we're in agreement on the new project choice.  @Eilon?